Source/Models/BrickDimension.swift

```diff
@@ -18,39 +18,106 @@ public struct BrickSize {
     }
 }
 
+public typealias RangeDimensionPair = (dimension: BrickDimension, minimumLength: CGFloat)
```

Comment:
We're already using `size` for `.fixed(size: 200)`? I think if we come up with the right terms, we should change it there too (might be some refactor on the WayfairApp though)...